data/json/professions.json

```diff
@@ -6322,5 +6328,50 @@
       "male": [ "briefs" ],
       "female": [ "panties", "bra" ]
     }
+  },
+  {
+    "type": "profession",
+    "id": "rv_hunter",
+    "name": "RV Hunter",
+    "description": "Ever since you were a child you loved hunting and traveling; so when you scraped enough money, you bought yourself an old rv, a gun, and hit the road.  You've been all over the country, hunting for good game.  Unfortunately the country is all over this time, and even more unfortunately you've lost your ol' reliable in the chaos.  Now you gotta find a new reliable.",
```

Comment:
```suggestion
    "description": "Ever since you were a child you loved hunting and traveling; so when you scraped enough money, you bought yourself an old RV, a gun, and hit the road.  You've been all over the country, hunting for good game.  Unfortunately the country is all over this time, and even more unfortunately you've lost your ol' reliable in the chaos.  Now you gotta find a new reliable.",
```